### ISSUE TITLE
At Cannot find module './BuckIcon'

### ISSUE THREAD

**Issue by mictadlo** [atom/atom#10674]
https://github.com/facebook/nuclide/issues/352
1. ppa:webupd8team/atom
2. sudo apt-get install atom
3. apm install nuclide
4. atom

**Atom Version**: 1.4.0
**System**: Linux Mint 17.3
**Thrown From**: [nuclide-buck-toolbar](https://github.com/facebook/nuclide) package, v0.0.0

### Stack Trace

#Failed to activate the nuclide-buck-toolbar package

```
At Cannot find module './BuckIcon'

Error: Cannot find module './BuckIcon'
    at Module._resolveFilename (module.js:336:15)
    at Function.Module._resolveFilename (/opt/atom/resources/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (/opt/atom/resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/home/lorencm/.atom/packages/nuclide/pkg/nuclide/buck/toolbar/lib/Activation.js:17:17)
    at Module._compile (/opt/atom/resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .js] (/opt/atom/resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (/opt/atom/resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.activate (/home/lorencm/.atom/packages/nuclide/pkg/nuclide/buck/toolbar/lib/main.js:27:20)
    at Package.module.exports.Package.activateNow (/opt/atom/resources/app.asar/src/package.js:183:20)
    at /opt/atom/resources/app.asar/src/package.js:156:32
    at Package.module.exports.Package.measure (/opt/atom/resources/app.asar/src/package.js:92:15)
    at /opt/atom/resources/app.asar/src/package.js:149:26
    at Package.module.exports.Package.activate (/opt/atom/resources/app.asar/src/package.js:146:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/opt/atom/resources/app.asar/src/package-manager.js:524:21)
    at /opt/atom/resources/app.asar/src/package-manager.js:505:29
    at Config.module.exports.Config.transactAsync (/opt/atom/resources/app.asar/src/config.js:337:18)
    at PackageManager.module.exports.PackageManager.activatePackages (/opt/atom/resources/app.asar/src/package-manager.js:500:19)
    at Object.activate (/home/lorencm/.atom/packages/nuclide/lib/main.js:215:31)
    at Package.module.exports.Package.activateNow (/opt/atom/resources/app.asar/src/package.js:183:20)
    at /opt/atom/resources/app.asar/src/package.js:156:32
    at Package.module.exports.Package.measure (/opt/atom/resources/app.asar/src/package.js:92:15)
    at /opt/atom/resources/app.asar/src/package.js:149:26
    at Package.module.exports.Package.activate (/opt/atom/resources/app.asar/src/package.js:146:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/opt/atom/resources/app.asar/src/package-manager.js:524:21)
    at /opt/atom/resources/app.asar/src/package-manager.js:505:29
    at Config.module.exports.Config.transactAsync (/opt/atom/resources/app.asar/src/config.js:337:18)
    at PackageManager.module.exports.PackageManager.activatePackages (/opt/atom/resources/app.asar/src/package-manager.js:500:19)
    at PackageManager.module.exports.PackageManager.activate (/opt/atom/resources/app.asar/src/package-manager.js:483:46)
    at AtomEnvironment.module.exports.AtomEnvironment.startEditorWindow (/opt/atom/resources/app.asar/src/atom-environment.js:688:21)
    at module.exports (/opt/atom/resources/app.asar/src/initialize-application-window.js:28:10)
    at setupWindow (file:///opt/atom/resources/app.asar/static/index.js:86:5)
    at window.onload (file:///opt/atom/resources/app.asar/static/index.js:41:9)
```

### Commands

```
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "tree-view"
    ]
  }
}
```

### Installed Packages

```coffee
# User
nuclide, v0.115.0

# Dev
No dev packages
```

**Comment by matthewwithanm** [comment]
Sorry, this is our (Nuclide's) fault. It was originally reported in facebook/nuclide#349 and fixed by facebook/nuclide@062aa89. We'll have a new release out soon!

**Comment by benogle** [comment]
:heart: @matthewwithanm
